grunttasks/server.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (grunt) {
 
   grunt.config('nodemon', {
     dev: {
-      script: 'bin/server.js',
+      script: 'bin/server.js','bin/internal.js',
```

Comment:
Does this actually parse? This is adding the `'bin/internal.js'` as an additional property of the `dev` object, not as part of `script`. Should this be wrapped in an array? `script: ['bin/server.js', 'bin/internal.js']`?